### DISCUSSION TITLE
Starting all plugins when closing the workspace looks strange

### DISCUSSION THREAD

**Discussion by lmcbout** [eclipse-theia/theia#7885] [General] [answered]
I noticed that when you close a workspace, on the console, we start all plugins. Why?
Should we activate the plugins when  we open a workspace? It looks weird to see the traces about all started pliugins when you close a workspace

**Comment by akosyakov** [comment]
Could you convert it into the issue and provide steps to reproduce?

**Reply by lmcbout** [reply]
Continue on  a new issue #7895

**Comment by akosyakov** [comment, marked as answer]
please see https://github.com/eclipse-theia/theia/issues/7895#issuecomment-634029835
